genestack_client/scripts/genestack_uploader.py

```diff
@@ -53,8 +53,12 @@
                    help='path to files or folders',
                    metavar='<paths>', nargs='+')
 group.add_argument('-n', '--no-recognition', help="don't try to recognize files", action='store_true')
-group.add_argument('-F', '--folder_name', metavar='<name>',
-                   help='name of the upload folder, if name is not specified it will be generated')
+
+exclusive_group = group.add_mutually_exclusive_group()
+exclusive_group.add_argument('-F', '--folder_name', metavar='<name>',
```

Comment:
Maybe it's time to fix `--folder_name`?

Reply:
It is part of public script API. It is not time to do it in this PR.